### ISSUE TITLE
Fix S6825 FP: Should not trigger on hidden input elements

### ISSUE THREAD

**Issue by cbaerike** [SonarSource/sonar-dotnet#8850]
### Description
Rule S6825 is supposed to trigger for aria-hidden="true", but also triggers for <input type="hidden">, which is incorrect.

Hidden input elements are often used in form-heavy applications to e.g. carry IDs for entities on edit screens. 

### Repro steps
Currently, the rule triggers on this code, if used in a *.cshtml file in a .NET MVC app:
```
 <input type="hidden" asp-for="ElementId" />
```

### Expected behavior
This code should not trigger the rule.

### Actual behavior
Rule S6825 triggers.

### Known workarounds
None

### Related information

* C# in .NET 8
* Visual Studio 2022
* dotnet build via Azure DevOps pipeline
* SonarCloud

**Comment by sebastien-marichal** [comment]
Hello @cbaerike,

Thank you for reporting the issue.

[S6825](https://sonarsource.github.io/rspec/#/rspec/S6825/html) is a rule introduced in [sonar-html](https://github.com/SonarSource/sonar-html), not in sonar-dotnet.
From my understanding, this seems to be a false positive.

I have created an issue in the sonar-html team backlog so they can tackle this issue later.
You can track it here: https://sonarsource.atlassian.net/browse/SONARHTML-219.
